amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/AmqpBrokerService.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.streamnative.pulsar.handlers.amqp;
+
+import lombok.Getter;
+import org.apache.pulsar.broker.PulsarService;
+
+/**
+ * AMQP broker related.
+ */
+public class AmqpBrokerService {
+    @Getter
+    private AmqpTopicManager amqpTopicManager;
+    @Getter
+    private ExchangeContainer exchangeContainer;
+    @Getter
+    private QueueContainer queueContainer;
```

Comment:
You aren't ever calling the getter for this.  You can remove the `@Getter` annotation.

Same for `ExchangeService` and `QueueService`.

Reply:
They are used in the constructor of AmqpChannel.java

Reply:
But they don't necessarily need to be exposed with getters if they aren't used anywhere.  It's a minor issue, and not a big deal.